app/assets/javascripts/govuk-component/tasklist.js

```diff
@@ -177,45 +177,45 @@
         event.preventDefault();
       }
 
-      function bindToggleOpenCloseAllButton(tasklistTracker) {
-        $openOrCloseAllButton = $element.find('.js-step-controls-button');
-        $openOrCloseAllButton.on('click', function () {
-          var shouldOpenAll;
+      function bindToggleShowHideAllButton(tasklistTracker) {
+        $showOrHideAllButton = $element.find('.js-step-controls-button');
+        $showOrHideAllButton.on('click', function () {
+          var shouldshowAll;
 
-          if ($openOrCloseAllButton.text() == bulkActions.openAll.buttonText) {
-            $openOrCloseAllButton.text(bulkActions.closeAll.buttonText);
-            $element.find('.js-toggle-link').text(bulkActions.closeAll.linkText)
-            shouldOpenAll = true;
+          if ($showOrHideAllButton.text() == bulkActions.showAll.buttonText) {
```

Comment:
not blocking but `$showOrHideAllButton.text()` feels weird to be using as the logic to open and close, I'd expect the instance of this component to have some state independent from the DOM.

`this.openSteps = 'all'` type deal